### DISCUSSION TITLE
Versionsnummer vergleichen

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#41] [Q&A] [answered]
Hej,

weißt du wie ich im Datenpunkt Stil zwei DPs vergleich kann (Onlineversion und Gerätversion) und wenn es die selbe ist grün machen und wenn nicht rot?

Beispiel:
`javascript.0.Tasmota.tasmota.version` (Hier habe ich mir ein kleines js-Skript gebaut womit ich mir über die github api die aktuelle master Version vom Tasmota git ziehe)
`sonoff.0.Arbeitszimmer.INFO.Info1.Version` (Tasmotaversion meiner Lichtschalter)

`sonoff.0.Arbeitszimmer.INFO.Info1.Version` diesen DP ziehe ich mir in jarvis und müsste ja dan glaube ich `val` sein oder?

Diese beiden DPs würde ich gerne vergleich. Hast du dazu was im docu?

VG
Florian

**Comment by mcuiobroker** [comment, marked as answer]
Und der Anzeige DP ist unterschiedlich?
Type string?
```
{"'{javascript.0.Tasmota.tasmota.version}'='{sonoff.0.Arbeitszimmer.INFO.Info1.Version}'":{"color":"green"},"default":{"color":"red"}}

{"'{javascript.0.Tasmota.tasmota.version}'='{val}'":{"color":"green"},"default":{"color":"red"}}
```

**Reply by Mephisto090** [reply]
Danke hat super funktioniert.


> Und der Anzeige DP ist unterschiedlich?

Was meinst du damit?

**Comment by mcuiobroker** [comment]
Wenn einer der DPs der Anzeige-DP ist, kann man die Variante mit `{val}` für den Anzeige-DP nutzen.

**Reply by Mephisto090** [reply]
Habe es jetzt mit diesem hier umgesetzt 

`{"'{javascript.0.Tasmota.tasmota.version}'='{val}'":{"color":"green"},"default":{"color":"red"}}`

**Reply by mcuiobroker** [reply]
ok
